tensorrt_llm/_torch/compilation/multi_stream/auto_multi_stream.py

```
@@ -0,0 +1,455 @@
+        "purple", "brown"
+    ]
+    with open(f"{call_cnt}.dot", 'w') as f:
+        f.write("digraph G {\n")
+        f.write(
+            f"id_entry [label=\"node=entry, distance={dag.entry_node.distance}\"]\n"
+        )
+        cnt = 0
+        for node in dag.nodes.values():
+            color = "white" if node.stream is None else COLORS[node.stream.id]
+            f.write(
+                f"id_{dag.node_to_id[node.node]} [label=\"node={node.node}, distance={node.distance}, weight={node.weight}\", color={color}, shape=oval]\n"
```

Ruff (0.12.2)

Line too long (155 > 120)

(E501)